source/minar.cpp

```diff
@@ -251,7 +251,7 @@ int minar::SchedulerData::start(){
 
             if(dispatch_tree.get_num_elements() > 0) {
                 CallbackNode *root = dispatch_tree.get_root();
-                now_plus_tolerance = wrapTime(now + root->tolerance);
+                now_plus_tolerance = wrapTime(now + root->tolerance/2);
```

Comment:
It is a bit weird that initially we're doubling the tolerance, only to divide it by 2 later. Being consistent is probably a better idea.